resources/timericon.js

```diff
@@ -188,18 +188,18 @@ class TimerIcon extends HTMLElement {
     this._textColor = 'white';
     this._colorBorderSize = 2;
 
-    if (this.duration != null) this._duration = Math.max(parseFloat(this.duration), 0);
-    if (this.width != null) this._width = Math.max(parseInt(this.width), 1);
-    if (this.height != null) this._height = Math.max(parseInt(this.height), 1);
-    if (this.bordercolor != null) this._borderFg = this.bordercolor;
-    if (this.bordersize != null) this._colorBorderSize = Math.max(parseInt(this.bordersize), 0);
-    if (this.scale != null) this._scale = Math.max(parseFloat(this.scale), 0.01);
-    if (this.hideafter != null && this.hideafter != '') this._hideAfter = Math.max(parseFloat(this.hideafter), 0);
+    if (this.duration !== null) this._duration = Math.max(parseFloat(this.duration), 0);
+    if (this.width !== null) this._width = Math.max(parseInt(this.width), 1);
+    if (this.height !== null) this._height = Math.max(parseInt(this.height), 1);
+    if (this.bordercolor !== null) this._borderFg = this.bordercolor;
+    if (this.bordersize !== null) this._colorBorderSize = Math.max(parseInt(this.bordersize), 0);
+    if (this.scale !== null) this._scale = Math.max(parseFloat(this.scale), 0.01);
+    if (this.hideafter !== null && this.hideafter !== '') this._hideAfter = Math.max(parseFloat(this.hideafter), 0);
     if (typeof this.onhide !== 'undefined') this._onHide = this.onhide;
```

Comment:
This isn't your code, but it is a === issue, in that these all should be null, and `!== undefined` is incorrect.